docs/installation/production/manual/configuration/basic/static_and_workspaces.rst

```diff
@@ -72,20 +72,7 @@ The app workspaces directory is one location where all app workspaces are collec
 
         Replace ``<TETHYS_WORKSPACES_ROOT>`` with the value returned by the previous command (see step 2.1).
 
-3. Collect the app workspaces to the ``TETHYS_WORKSPACES_ROOT`` location:
-
-    .. code-block::
-
-        tethys manage collectworkspaces
-
```

Comment:
Things like this should probably be marked as deprecated instead of outright removed.

Reply:
Added this back in with a warning that it is deprecated and not needed with using the `Paths API`